tools/mot_error_visualization.py

```diff
@@ -0,0 +1,266 @@
+import argparse
+import os
+import os.path as osp
+
+import cv2
+import mmcv
+import motmetrics as mm
+import numpy as np
+import seaborn as sns
+from mmcv import Config
+
+from mmtrack.datasets import build_dataset
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='visualize the error for multiple object tracking')
+    parser.add_argument('config', help='path of the config file')
+    parser.add_argument('--result-file', help='path of inference result')
+    parser.add_argument(
+        '--out-dir',
+        help='directory where painted images or videos will be saved')
+    parser.add_argument(
+        '--out-video', action='store_true', help='whether to output video')
+    parser.add_argument(
+        '--out-image', action='store_true', help='whether to output image')
+    parser.add_argument(
+        '--show',
+        action='store_true',
+        help='whether to show the results on the fly')
+    parser.add_argument(
+        '--fps', type=int, default=3, help='FPS of the output video')
+    args = parser.parse_args()
+    return args
+
+
+def show_wrong_tracks(img,
```

Comment:
Put this function to `visualization.py` in `mmtrack/core/utils`, and support different `backends`.